### PR TITLE
chore: visual test for materials

### PR DESCRIPTION
**Pull request by AjimenezDCL**
#144

### CODE REVIEW

**Review by Suduck — APPROVED** [review]

**Comment by menduz** [comment]
Screen Shot 2021-07-16 at 14 48 25
Are the changes in the default environment parameters intentional?

**Comment by AjimenezDCL** [comment]
> Screen Shot 2021-07-16 at 14 48 25
> Are the changes in the default environment parameters intentional?

@menduz 
Yes! 
The lightning settings in our visual tests scene were completly different from our world scene. Now the visual tests should give a better representation of the real environment. 

You can check the changes in the file: `unity-renderer/Assets/Scenes/Test/MainVisualTest.unity` to get a better grasp of what has been changed.

Additionally we were not applying post-processing to the screenshots. It was not a problem until we started testing emission (bloom) and SSAO.

Therefore, our previous visual tests are not valid anymore.

**Review by Suduck — APPROVED** [review]

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Materials/Tests/PBRMaterialVisualTests.cs [hunk 1]
```diff
@@ -9,16 +11,12 @@
 
 public class PBRMaterialVisualTests : VisualTestsBase
 {
-    [UnityTest]
-    [VisualTest]
-    [Explicit]
-    [Category("Explicit")]
+    [UnityTest, VisualTest]
+    [Explicit, Category("Explicit")]
```
Comment (pravusjif) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Materials/Tests/PBRMaterialVisualTests.cs:15]: should we still keep these as explicit?

**Review by pravusjif — APPROVED** [review]

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Materials/Tests/PBRMaterialVisualTests.cs (hunk 1 above)
Comment (AjimenezDCL) [reply unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Materials/Tests/PBRMaterialVisualTests.cs:15]: These are legacy tests, it was not the goal of this PR to assert the state of the old ones.

Comment (AjimenezDCL) [reply unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Materials/Tests/PBRMaterialVisualTests.cs:15]: These are legacy tests, it was not the goal of this PR to assert the state of the old ones.

**Pull request merged**